components/Account/NewsletterSubscriptions.js

```diff
@@ -80,7 +98,59 @@ const NewsletterSubscriptions = props => (
         <Fragment>
           {status !== 'subscribed' && (
             <Box style={{ margin: '10px 0', padding: 15 }}>
-              <P>{t('account/newsletterSubscriptions/unsubscribed')}</P>
+              <Mutation mutation={RESUBSCRIBE_EMAIL}>
+                {(mutate, { loading, error, data: mutationData }) => (
+                  <>
+                    {status === 'unsubscribed' && (
+                      <P>{t('account/newsletterSubscriptions/unsubscribed')}</P>
+                    )}
+                    {/* Show if the status has been set to pending */}
+                    {!error &&
+                      mutationData?.resubscribeEmail?.status === 'pending' && (
```

Comment:
Is the `!error` check here really necessary? I feel like this could be removed to simplify things.

Reply:
You're right, one can assume error to be undefined if mutationData is defined.